### PR TITLE
New zero state for Toggler

### PR DESCRIPTION
**Pull request by joeyyang**
This PR makes the following change:
- Creates new zero state screen to match other plugins' zero state designs

Screen Shot 2022-12-05 at 10 38  41AM

joey-zero-state

### CODE REVIEW

**Review by lynnerang — APPROVED** [review]
Nice nice

File: toggler/src/views/ThemeToggler.svelte [hunk 1]
```diff
@@ -54,16 +54,21 @@
     left: 0;
     right: 0;
     bottom: 0;
-    background-color: var(--canvas-bg-overlay);
+    background-color: var(--white-100o);
```
Comment (gblakeman) [inline toggler/src/views/ThemeToggler.svelte:57]: Have you tested this update in Figma’s Dark Mode? The idea of base color tokens like `--white-100o` is that they are _not_ used within components. We only use semantic tokens in the component. Those semantic tokens are aliased to base tokens in our colors file based on the light/dark theme needs.

**Review by gblakeman — COMMENTED** [review]
Looks good, but questions around Dark Mode testing.

**Comment by joeyyang** [comment]
thanks @gblakeman - please take a look!

**Review by gblakeman — COMMENTED** [review]
I think I forgot to get back to this – Dark theme looks great; thanks for those changes!

Can you do the SVG mark inline so that the implementation matches our other plugins? Putting it in an `img` still won’t allow us to use CSS to adjust it in the future.

**Comment by joeyyang** [comment]
@gblakeman update made, please take a look

**Comment by joeyyang** [comment]
Screen Shot 2022-12-19 at 2 24  12PM

@gblakeman received your feedback - I would still rather provide the correct instruction the first time, so I've simplified the copy to the above. Let me know what you think~

**Review by gblakeman — APPROVED** [review]
🛳💥

**Pull request merged**